### PR TITLE
update(exploring-rx-operators-map): clarified code snippets and text

### PR DESCRIPTION
**Pull request by ThomasBurleson**
* clarified some code snippets
* add some more verbiage regarding Observables
* add content why `.map()` is the basic operator and builds from there...

### CODE REVIEW

File: _posts/2016-05-16-exploring-rx-operators-map.md [hunk 1]
```diff
@@ -86,11 +91,14 @@ let obs = Rx.Observable.fromEvent(demoInput, 'input')
...
 {% endhighlight %}
 
-Of course we could have done the same in the first `map` call already but it's sometimes more readable to break things into multiple steps. We also often use different operators in between of two `map` calls (e.g to filter something out).
+Of course, we could have done the same in the first `map` call. But it's sometimes more readable to break things into multiple steps. Notice that often we also use different operators in between of two `map` calls (e.g to filter something out).
 
 If you like to play a bit with the operator yourself, here is a working demo.
 
 <iframe src="http://embed.plnkr.co/iWR9b2s4wd0pZSOEfHuR/"></iframe>
 
+At this point, you may think that Observables are really just a minor enhancement on the Observer or Futures patterns... better suited to handle a sequence of events rather than a single callback. And the `.map()` function certainly does not - at first glance - seem to offer any added-value. The power of Observables is revealed when you start using Rx operators to transform, combine, manipulate, and work with sequences of items emitted by Observables.
```
Comment (ThomasBurleson) [inline _posts/2016-05-16-exploring-rx-operators-map.md]: Should be `the Observer or Promise patterns...`

File: _posts/2016-05-16-exploring-rx-operators-map.md [hunk 2]
```diff
@@ -77,6 +81,7 @@ We can go on and chain `map` calls to project the data even further. For instanc
 {% raw %}
 let obs = Rx.Observable.fromEvent(demoInput, 'input')
                        .map(e => e.target.value)
+					   .filter( value => value > 100 )
```
Comment (ThomasBurleson) [inline _posts/2016-05-16-exploring-rx-operators-map.md]: I need to confirm the spacing on this.

File: _posts/2016-05-16-exploring-rx-operators-map.md (hunk 1 above)
Comment (PascalPrecht) [inline _posts/2016-05-16-exploring-rx-operators-map.md]: Correct, "Futures" might confuse ppl that are not coming from Java land or other languages where this terminology is a thing

File: _posts/2016-05-16-exploring-rx-operators-map.md (hunk 2 above)
Comment (PascalPrecht) [inline _posts/2016-05-16-exploring-rx-operators-map.md]: I think we should rather go with:

```
...fromEvent()
  .map()
  .filter()
```

@cburgdorf ?

**Pull request merged**